canvas_sdk/v1/data/questionnaire.py

```diff
@@ -0,0 +1,165 @@
+from django.db import models
+
+from canvas_sdk.v1.data import Patient
+from canvas_sdk.v1.data.base import CommittableModelManager
+from canvas_sdk.v1.data.user import CanvasUser
+
+
+class ResponseOptionSet(models.Model):
+    """ResponseOptionSet."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_responseoptionset_001"
+
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
```

Comment:
Maybe we should start creating some mixins for timestamped models and audited models so we don't keep repeating ourselves?

Reply:
I can be convinced, but I think having them all listed explicitly here makes them more reference-able for developers. The mixin is important when defining managed models so you end up with consistent naming and types, but for these unmanaged models all it does is obfuscate the attribute's presence and definition.

Reply:
Fair enough